### PR TITLE
fr-command-rar: parse date & time using strptime

### PR DESCRIPTION
**Pull request by rbuj**
test:
1 - install rar in /usr/local
```
$ wget https://www.rarlab.com/rar/rarlinux-x64-612.tar.gz
$ tar xzvf rarlinux-x64-612.tar.gz 
$ cd rar
$ sudo make install
```
2 - create test.rar archive
```
$ rar a test.rar rar
```
3 - open test.rar
```
$ engrampa test.rar
```

### CODE REVIEW

File: src/fr-command-rar.c [hunk 1]
```diff
@@ -20,6 +20,8 @@
  *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+#define _XOPEN_SOURCE 700
```
Comment (zhuyaliang) [inline src/fr-command-rar.c:23]: Why 700

Comment (L-U-T-i) [reply src/fr-command-rar.c:23]: https://stackoverflow.com/questions/5378778/what-does-d-xopen-source-do-mean

Comment (zhuyaliang) [reply src/fr-command-rar.c:23]: Thank you. This answers my doubts

**Review by zhuyaliang — APPROVED** [review]
Simple and safe code is conducive to maintenance

**Review by lukefromdc — COMMENTED** [review]
Builds and runs fine

**Pull request merged**